components/Modal/ReadMe.md

```diff
@@ -7,16 +7,6 @@ const visible = ref(false)
 
 # Modal
 
-<DemoWrapper>
-	<Modal v-model:show="visible" title="modal">
```

Comment:
@elevatebart I removed this modal at the top because it was:
1. Redundant with the other modals.
2. Not actually working properly (the styles would not update for some reason).